src/ppl/src/IOPlacer.tcl

```diff
@@ -359,13 +359,14 @@ sta::define_cmd_args "place_pin" {[-pin_name pin_name]\
                                   [-layer layer]\
                                   [-location location]\
                                   [-pin_size pin_size]\
-                                  [-force_to_die_boundary]
+                                  [-force_to_die_boundary]\
+                                  [-placed_status]
 }
 
 proc place_pin { args } {
   sta::parse_key_args "place_pin" args \
   keys {-pin_name -layer -location -pin_size}\
-  flags {-force_to_die_boundary}
+  flags {-force_to_die_boundary -placed_status}
```

Comment:
@eder-matheus
Should this flag be documented? Can you make a PR or provide the info to @luarss ?

Reply:
It will not be necessary. I'm about to remove this flag since it isn't working as I expected and we will not need it anymore.